src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -267,18 +267,16 @@ public void parse_missingConfigValue_throwsParseException() throws ParseExceptio
         ArgsParser.parse(translateCommandline(input));
     }
 
-    @Test(expected = ParseException.class)
-    public void file_forOutputDirectory_throwsParseException() throws ParseException {
-        String file = PROJECT_DIRECTORY.resolve("parser_test.csv").toString();
-        String input = DEFAULT_MANDATORY_ARGS + String.format("-output %s", file);
-        ArgsParser.parse(translateCommandline(input));
-    }
-
-    @Test(expected = ParseException.class)
-    public void nonExistentDirectory_forOutputDirectory_throwsParseException() throws ParseException {
+    @Test
+    public void nonExistentDirectory_forOutputDirectory_createDirectory() throws ParseException, IOException {
```

Comment:
misaligned to @yamidark suggested nit

>Slight nit:
outputPath_nonExistentDirectory_success()
The naming convention for test methods we use is featureUnderTest_testScenario_expectedBehaviour()

Reply:
Here our 
featureUnderTest -> `outputPath
testScenario -> `nonExistentDirectory`
expectedBehaviour/outcome -> `success`

Reply:
ok, test method name has been updated accordingly